docs/content/Aggregations.md

```diff
@@ -46,12 +46,28 @@ Computes the sum of values as 64-bit floating point value. Similar to `longSum`
 
 #### `max` aggregator
 
-`max` computes the maximum metric value
+`max` computes the maximum metric value and returns a double.
 
 ```json
 { "type" : "max", "name" : <output_name>, "fieldName" : <metric_name> }
 ```
 
+#### `longMin` aggregator
+
+`longMin` computes the minimum metric value and returns a long.
```

Comment:
Consider documenting that it will return the minimum among `Long.MAX_VALUE` and the metric values